content/smart.md

```diff
@@ -154,7 +156,17 @@ SMART apps can be embedded and launched from the *MPages* Workflow or Summary. I
 
 #### iframe Cookies ####
 
-If your SMART app uses cookies, the [P3P](https://www.w3.org/P3P/) header or meta tag must be set in order for cookies to be saved correctly while running in an *MPages* iframe.  Please see [this page](https://blogs.msdn.microsoft.com/ieinternals/2013/09/17/a-quick-look-at-p3p/) for more information on P3P.
+For IE if your SMART app uses cookies, the [P3P](https://www.w3.org/P3P/) header or meta tag must be set in order for cookies to be saved correctly while running in an *MPages* iframe.  Please see [this page](https://blogs.msdn.microsoft.com/ieinternals/2013/09/17/a-quick-look-at-p3p/) for more information on P3P.
+
+For Edge if your SMART app uses cookies, the `Set-Cookie` response header has to contain the attributes: `SameSite=None; Secure` for your cookies to be saved correctly while running in an iframe.
+
+### Microsoft Edge Browser (Chromium based) Support ###
+We are excited to share that there is full support for embedded Microsoft Edge browser (Chromium based) inside *Cerner Millennium PowerChart*!
```

Comment:
`We are excited to share that there is full support for embedded Microsoft Edge browser (Chromium based) inside *Cerner Millennium PowerChart*!`

Usually this sort of wording goes in as part of a flash notification to user's where we are announcing the functionality. 
Another aspect would be to not use words such as `full support` since we are not yet aware of the hurdles that a user might face when they start launching their apps against Chromium.

Since these are our docs we could word it as follows :
`Along with Internet Explorer, Cerner now supports embedded Microsoft Edge browser (Chromium based) inside *Cerner Millennium PowerChart*`